.stripesclirc.js

```diff
@@ -76,19 +76,19 @@ const testPlugin = {
         return rule.loader === 'babel-loader';
       });
 
+      // Make decorators possible
+      config.module.rules[babelLoaderConfigIndex].options.plugins = [
```

Comment:
Won't this one also override the babel loader plugins?

Reply:
You're right. `stripes-core` didn't have any `babel-loader` plugins, but I just bulletproofed it in case it does someday.